client/src/components/ElementEditor/Content.js

```diff
@@ -1,37 +1,55 @@
 import React, { PureComponent, PropTypes } from 'react';
+import { inject } from 'lib/Injector';
 
 class Content extends PureComponent {
   render() {
-    const { fileUrl, fileTitle, content } = this.props;
+    const {
+      fileUrl,
+      fileTitle,
+      content,
+      previewExpanded,
```

Comment:
It feels like this should be a context argument

E.g. `if (context === 'preview') { return <SummaryComponent ...props />; } else { return <FormBuilderComponent ...props />}`

Reply:
I see - this is loosely coupled to the container's state name... leave it as is +D